src/Appwrite/Utopia/Response.php

```diff
@@ -251,6 +254,12 @@ class Response extends SwooleResponse
     public const MODEL_BUILD_LIST = 'buildList';  // Not used anywhere yet
     public const MODEL_FUNC_PERMISSIONS = 'funcPermissions';
     public const MODEL_HEADERS = 'headers';
+    public const MODEL_FUNCTION_TEMPLATE = 'functionTemplate';
+    public const MODEL_FUNCTION_TEMPLATE_LIST = 'functionTemplateList';
+    public const MODEL_RUNTIME_TEMPLATE = 'runtimeTemplate';
+    public const MODEL_VARIABLE_TEMPLATE = 'variableTemplate';
+    public const MODEL_RUNTIME_TEMPLATE_LIST = 'runtimeTemplateList';
+    public const MODEL_VARIABLE_TEMPLATE_LIST = 'variableTemplateList';
```

Comment:
what are these for?

Reply:
The templates have a nested structure, like:
template1:
- runtimes
- variables

So had to add models for runtimes and variables.